s3tests_boto3/functional/test_s3select.py

```diff
@@ -135,6 +150,53 @@ def create_random_csv_object_string(rows,columns,col_delim=",",record_delim="\n"
 
         return result
 
+def create_random_csv_object_trim(rows,columns,col_delim=",",record_delim="\n",csv_schema=""):
```

Comment:
it's not random object. 
all rows are identical, it's cause for wrong test coverage.